apps/drec-api/src/models/DeviceGroup.ts

```diff
@@ -1,8 +1,34 @@
-import { IDevice } from '.';
+import { IDevice } from './Device';
+import {
+  CapacityRange,
+  CommissioningDateRange,
+  Installation,
+  OffTaker,
+  Sector,
+  StandardCompliance,
+} from '../utils/enums';
 
 export interface IDeviceGroup {
   id: number;
   name: string;
   organizationId: number;
+  fuelCode: string;
+  countryCode: string;
+  standardCompliance: StandardCompliance;
+
+  deviceTypeCodes: string[];
+  offTakers: OffTaker[];
+  installationConfigurations: Installation[];
+  sectors: Sector[];
+  gridInterconnection: boolean; // True - all devices have gridInterconnection true, if one has false, then this value is false
```

Comment:
Why these values serve as a "read model" for devices under group?
Can't we just query devices and compute all these values?

Reply:
Device groups will act as 'devices' and we will use most of the properties to filter device groups. We will have performance issues to query this information from devices inside a device group. The decission is to expose them at the group level based on the devices.